media/css/firefox/challenge-the-default/_toggle.scss

```diff
@@ -5,85 +5,132 @@
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
 .toggle {
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-  flex-wrap: wrap;
-  position: relative;
-  padding-bottom: 0;
-  transform: rotate(0);
-
-  &.animate-slide {
-    transform: rotate(-30deg);
-    transition: transform 250ms 250ms ease-in
-  }
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+    position: relative;
+    padding-bottom: 0;
+
+    &.animate-slide {
+        animation: animate-toggle 800ms 1 linear forwards;
+    }
+}
+
+
+
+@keyframes animate-toggle {
+    0% {
+        transform: rotate(0);
+    }
+
+    25% {
+        transform: rotate(-5deg);
+    }
+
+    50% {
+        transform: rotate(-12deg);
+    }
+
+    75% {
+        transform: rotate(-5deg);
+    }
+
+    100% {
+        transform: rotate(0);
+    }
+}
+
+@keyframes animate-toggle-rotate {
+    0% {
+        transform: translate(0, -50%) rotate(0deg);
+
+    }
+
+    25% {
+        transform: translate(25%, -50%) rotate(0deg);
+    }
+
+    50% {
+        transform: translate(50%, -50%) rotate(-10deg);
+    }
+
+    75% {
+        transform: translate(75%, -50%) rotate(-20deg);
+    }
+
+    100% {
+        transform: translate(100%, -50%) rotate(0);
+    }
 }
 
 .toggle-display {
-  --offset: 0.25em;
-  --diameter: 2.5em;
-  display: flex;
-  align-items: center;
-  justify-content: space-around;
-  width: calc(var(--diameter) * 2 + var(--offset) * 2);
-  height: calc(var(--diameter) + var(--offset) * 2);
-  border: 1px solid $color-white;
-  position: relative;
-  border-radius: 100vw;
-  background-color: transparent;
-  transition: 250ms;
-  cursor: pointer;
-  pointer-events: none;
-
-  @media #{$mq-md} {
-    --diameter: 3em;
-  }
-
-  @media #{$mq-lg} {
-    --diameter: 3.5em
-  }
+    --offset: 0.25em;
+    --diameter: 2.5em;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: calc(var(--diameter) * 2 + var(--offset) * 2);
+    height: calc(var(--diameter) + var(--offset) * 2);
+    border: 1px solid $color-white;
+    position: relative;
+    border-radius: 100vw;
+    background-color: transparent;
+    transition: 800ms;
```

Comment:
Do they really want it this slow? It feels sluggish to me but if it's intentionally slow then so be it, just want to make sure it wasn't a mistake.